src/autopas/pairwiseFunctors/Functor.h

```diff
@@ -223,6 +224,18 @@ class Functor {
    */
   double getCutoff() const { return _cutoff; }
 
+  /**
+   * Get the number of FLOPs. Implementation required if FLOP logger used.
+   * @return number of FLOPs
+   */
+  virtual size_t getNumFLOPs() { return 0; }
```

Comment:
```suggestion
  virtual size_t getNumFLOPs() const { return 0; }
```
We should be able to make this and `getHitRate()` both a `const` function, right?

Reply:
Done in 7fa81126